projects/ui-components/lf-user-feedback/feedback-image-upload/feedback-image-upload.component.html

```diff
@@ -2,7 +2,7 @@
   >{{ localizedStrings.ATTACH_IMAGE | async }} ({{ localizedStrings.OPTIONAL | async }})</label
 >
 <div (drop)="dropHandler($event)" (dragover)="dragOverHandler($event)">
-  <div id="image-upload" class="file-drop-zone" *ngIf="!imageUploaded; else showSelectedFile">
+  <div id="image-upload" class="file-drop-zone" *ngIf="!(imageUploaded || rawImageBase64); else showSelectedFile">
```

Comment:
why do we need to check whether rawImageBase64 exists - shouldn't it be sufficient to check imageUploaded?

Reply:
We actually wanted to check rawImageBase64, because that is what we have to send in the request. 
These variables had the same "lifecycle" and were assumed to be in sync (should be undefined, defined at the same time), but they weren't due to a bug. That's a warning sign. I ended up combining the name (all that we needed from the file) and the base64 into one variable which makes more sense than having two variables we assume are in sync, because two variables makes it more probable that there are bugs due to the variables not being in sync (as this bug was)